docs/source/guide/ddd-2-use-case.myst

```diff
@@ -12,3 +12,20 @@ kernelspec:
 ---
 
 # When should I use DDD?
+
+## Advantages
+
+Dynamical decoupling is a technique that has been originally devised to shield a quantum system from quantum noise coming from an environment, using open-loop control techniques :cite:`Viola_1999_PRL`. The crucial idea behind dynamical decoupling is to intervene with control pulses on timescales that are faster than those of the system-environment interaction.
+
+The theory of dynamical decoupling was originally devised at the pulse level and can include environments with memory effects. It can thus be effective also against structured baths or so called color-noise environments.
+
+Dynamical decoupling is also effective for one-shot quantum computing algorithms, i.e., it finds application beyond algorithms that just require expected values, like variational quantum algorithms.
```

Comment:
```suggestion
Dynamical decoupling is also effective for single-shot quantum computing algorithms, i.e., it finds application beyond algorithms that just require expectation values, like variational quantum algorithms.
```
Good idea adding this point. It's a very important advantage with respect to other techniques.